src/libData/AccountData/AccountStoreSC.tpp

```diff
@@ -695,6 +736,7 @@ bool AccountStoreSC<MAP>::ParseCallContractJsonOutput(const Json::Value& _json,
     }
     return false;
   }
+  uint64_t startGas(gasRemained);
```

Comment:
Minor comment, maybe change to `uint64_t startGas = gasRemained;` just for coding consistency